drivers/bridge/labels.go

```diff
@@ -0,0 +1,27 @@
+package bridge
+
+const (
+	// BridgeName label for bridge driver
+	BridgeName = "io.docker.network.bridge.name"
```

Comment:
I know other projects, such as Swarm and Docker Compose, are using `com.docker.<projectname>` as namespace for labels. For consistency, I think `com.docker` in stead of `io.docker` should be used. Possibly `libnetwork` as well, but less opinionated on that.


Should there be a constant defining the namespace as well?